client/containers/Pub/SpubHeader/SpubHeader.tsx

```diff
@@ -21,54 +21,45 @@ type Props = {
 	pubData: DefinitelyHas<PubPageData, 'submission'>;
 };
 
-export const renderInstructionTabTitle = (icon: IconName, title: string) => {
-	return (
-		<>
-			<Icon icon={icon} /> {title}
-		</>
-	);
-};
+export const renderTabTitle = (icon: IconName, title: string) => (
+	<>
+		<Icon icon={icon} /> {title}
+	</>
+);
 
 const SpubHeader = (props: Props) => {
-	const { pubData, historyData, updateLocalData } = props;
```

Comment:
Preference because a locally defined variable might be the result of mutations on a received prop, whereas passing straight from props lets someone looking at the instantiation of the `<Component prop1={props.x.y.z} />` know that they should look upstream to find the origin of the data in `z`, that it's not set in this file. Feels more function-y, less object/state-y.

I was relieved that typescript makes deep referencing safer (e.g. `props.x.y.z`) which made destructuring in order to check for presence/type safety unnecessary.

For me, the balance to strike is in passing smaller, flatter pieces of data into sub-components to minimize the number of `dots.we.have.to.traverse` to find the variable we need, but still refraining from destructing props that will be passed through unchanged.